src/EtwCollector/EtwTelemetryModule.cs

```diff
@@ -180,7 +180,7 @@ private void EnableProviders()
             {
                 ProviderGuid = TplActivities.TplEventSourceGuid,
                 Level = TraceEventLevel.Always,
-                Keywords = TplActivities.TaskFlowActivityIdsKeyword
+                Keywords = TplActivities.TaskFlowActivityIdsKeyword,
```

Comment:
is this a compilation error?

Reply:
not a complication error. seems like a code standards thing, they want to include the trailing comma  everywhere. 
i thought it's a very minor non-functional change so i did it everywhere.